src/_gettsim/parameters/ges_rente.yaml

```diff
@@ -975,6 +976,109 @@ abolishment_cohort_rente_für_arbeitsl:
   note: https://www.sozialgesetzbuch-sgb.de/sgbvi/237.html
   1980-01-01:
     scalar: 1952
+altergrenze_arbeitsl_abschlagsfrei_vertrauensschutz:
+  name:
+    de: >-
+      Altersgrenze für Altersrente wegen Arbeitslosigkeit mit  Vertrauensschutz
+      (abschlagsfrei).
+    en: >-
+      Age threshold for pension for unemployed (without deductions)
+      with Vertrauensschutz
+  description:
+    de: >-
+      Frühesmögliches Alter ab dem die Altersrente für Arbeitslose, für die der
+      Vertrauensschutz gilt abschlagsfrei bezogen werden kann.
+    en: >-
+      Earliest possible age to receive pension for unemployed without  deductions
+      under legitimate expectations.
+  reference: § 237 SGB VI
+  note: >-
+    https://www.gesetze-im-internet.de/sgb_6/__237.html
+  type: piecewise_linear
+  1980-01-01:
+    0:
+      lower_threshold: -inf
+      upper_threshold: 1940.916666666
+      rate_linear: 0
+      intercept_at_lower_threshold: 60
+    1:
+      upper_threshold: 1941.0
+      rate_linear: 1
+    2:
+      upper_threshold: 1941.25
+      rate_linear: 0.0
+    3:
+      upper_threshold: 1941.3333333
+      rate_linear: 1
+    4:
+      upper_threshold: 1941.583333333
+      rate_linear: 0.0
+    5:
+      upper_threshold: 1941.6666666
+      rate_linear: 1
+    6:
+      upper_threshold: 1941.9166666
+      rate_linear: 0.0
+    7:
+      upper_threshold: 1942.0
+      rate_linear: 1
+    8:
+      upper_threshold: 1942.25
+      rate_linear: 0.0
+    9:
+      upper_threshold: 1942.3333333
+      rate_linear: 1
+    10:
+      upper_threshold: 1942.583333333
+      rate_linear: 0.0
+    11:
+      upper_threshold: 1942.6666666
+      rate_linear: 1
+    12:
+      upper_threshold: 1942.9166666
+      rate_linear: 0.0
+    13:
+      upper_threshold: 1943.0
+      rate_linear: 1
+    14:
+      upper_threshold: 1943.25
+      rate_linear: 0.0
+    15:
+      upper_threshold: 1943.3333333
+      rate_linear: 1
+    16:
+      upper_threshold: 1943.583333333
+      rate_linear: 0.0
+    17:
+      upper_threshold: 1943.6666666
+      rate_linear: 1
+    18:
+      upper_threshold: 1943.9166666
+      rate_linear: 0.0
+    19:
+      upper_threshold: 1944.0
+      rate_linear: 1
+    20:
+      upper_threshold: 1944.08333333
+      rate_linear: 0.0
+    21:
+      upper_threshold: 1944.16666667 # 25  #  jump to 65 (normal unemployed)
```

Comment:
I neither get the comments nor the very high rate here.

Reply:
For people born after Feb 1944 Vertrauensschutz is no longer available, hence the regular thresholds for the deduction free pension for unemployed apply, which is 65 for this cohort.

Reply:
Yeah, that's exactly what is not intuitive then. If this is called Vertrauensschutz and only relevant for a couple of cohorts, others should not even show up here.

Reply:
yep, thats due to the "exhaustiveness requirement" of the function (needs inputs from -inf to inf)-  had this problem before. maybe the function should be changed then.

Reply:
I changed the parameter input now. it required a littlebit of hardcoding numbers in the function, as the threshold increase lost its regularity shortly before abolishment. hope thats fine, otherwise can change it